[docs] Improv ecosystem documentation (#78)
OpenBAS-Platform/docs: docs/deployment/overview.md

### Neutral agents / executors

Executors are embedded into the platform but you should configure at least one.

> It is the system that will be used to execute local injectors on endpoints.

Reviewer: ```suggestion
This system is responsible for executing local injectors on endpoints.
```
